src/main/java/entities/ComputerPostProcess.java

```diff
@@ -0,0 +1,23 @@
+package entities;
+
+import interfaces.Computer;
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+
+@Log4j
+public class ComputerPostProcess implements BeanPostProcessor {
+    public Object postProcessBeforeInitialization(Object bean, String beanName) throws BeansException {
```

Comment:
Is it possible that bean is null?

Reply:
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/config/BeanPostProcessor.html
It can be nullable, its what read already. What should I do here?

Reply:
Now it is ok.